src/controllers/pagoPAProxyController.ts

```diff
@@ -24,9 +25,10 @@ export default class PagoPAProxyController {
   public async getPaymentInfo(
     req: express.Request
   ): Promise<
+    | IResponseErrorGeneric
     | IResponseErrorInternal
```

Comment:
as you add additional responses to an endpoint, you must also update the API specs: https://github.com/teamdigitale/italia-backend/blob/master/api_proxy.yaml#L291